components/brave_wallet/browser/swap_controller.cc

```diff
@@ -117,6 +117,24 @@ GURL SwapController::GetTransactionPayloadURL(
   return url;
 }
 
+void SwapController::GetSwapConfiguration(
+    const std::string& chain_id,
+    GetSwapConfigurationCallback callback) {
+  auto swap_config = mojom::SwapConfig::New();
+
+  if (chain_id == mojom::kRopstenChainId) {
+    swap_config->swap_api_url = kRopstenSwapAPIURL;
+    swap_config->buy_token_percantage_fee = kRopstenBuyTokenPercentageFee;
+    swap_config->fee_recipient = kRopstenFeeRecipient;
+  } else {
```

Comment:
I think better to differentiate between a known and unknown swap config for a given chain id.

Reply:
It's just a copy from desktop https://github.com/brave/brave-core/pull/10215/files#diff-6dcef5d72bc94ad3e02ebfb7fb11418fb009cb9cb08dc34c52268d30066e1f6dR21. @onyb should I just return an empty object for other networks?

Reply:
Swap is disabled on all networks other than mainnet or ropsten. So the `else` case would never be used.